GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -45,6 +44,12 @@
 
 @implementation GDTCORStorage
 
++ (void)load {
```

Comment:
AFAIK, any code in `+ (void)load` method contributes to the app startup time. I wonder, may we defer storage registration to later stages, ideally until it's actual first use.

Reply:
+1

Reply:
In this case, nothing will directly refer to GDTCORFlatFileStorage and so `+initialize` (the next-best option) will never be called.